UI/src/view/MassTransitStopLayer.js

```diff
@@ -588,6 +588,10 @@ window.MassTransitStopLayer = function(map, roadCollection, mapOverlay, assetGro
         renderAssets(groupedAssets);
       }
     });
+    eventListener.listenTo(eventbus, 'toggleWithRoadAddress', function(){
+      if(applicationModel.getSelectedLayer() == layerName)
+        me.refreshView();
+    });
```

Comment:
In this case, it's more pretty if we create a function bellow and not at eventListener, only for use the same idea the others "listenTo's".